src/app/components/SwapSubmitModal/view.jsx

```diff
@@ -58,9 +59,13 @@ const SwapSubmitModal = ({
           ? display.fiat(swundle.totalTxFee)
           : <Spinner inline size='sm'/>}
         </p>
+        {(secondsUntilPriceExpiry > 0 && showTimer)
```

Comment:
These nested conditions would be clearer as
```
{showTimer && (
  secondsUntilPriceExpiry > 0 
    ? (...)
    : (...)
)}
```

Reply:
do you mean putting showTimer first, or getting rid of the second ternary? without the `!showTimer ? (...) : null` if you open the order modal after the timer expires, until the swaps are received it will show the message about how the 15 minutes window has closed